org.godotengine.Godot.json

```diff
@@ -0,0 +1,117 @@
+{
+    "app-id": "org.godotengine.Godot",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "branch": "stable",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "godot",
+    "tags": [],
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* Network access */
+        "--share=network",
+        /* Sound access */
+        "--socket=pulseaudio",
+        /* Wayland access */
+        "--socket=wayland",
+        /* We want full fs access so we can read the files */
+        "--filesystem=host",
+        "--device=dri"
+    ],
+    "build-options" : {
```

Comment:
Shouldn't we fallback to upstream CFLAGS? If not, wouldn't it make more sense to have these enforced by flatpak-builder (even with some command line parameter) so we could stop duplicating it everywhere?

Reply:
This is a tricky area. I like the build result to be repeatable, but having the cflags as a commandline means you will not get the same build from two runs. Also, such a commandline would just be a default, we would still require it to be possible to override for some modules, and we have no great way of doing that for a globally set thing atm.

It is kind of painful to have to repeat this all over the place though.

I'm not sure there is a great "default" in upstream, i think it just does nothing. Will check.